src/ore/src/tracing/mod.rs

```diff
@@ -368,7 +369,35 @@ where
     let stack = stack.with(otel_layer);
     #[cfg(feature = "tokio-console")]
     let stack = stack.with(tokio_console_layer);
-    let stack = stack.with(sentry_tracing::layer().event_filter(sentry_event_filter));
+    let stack = stack.with(
+        sentry_tracing::layer()
+            .event_filter(sentry_event_filter)
+            // WARNING, ENTERING THE SPOOKY ZONE
+            //
+            // While sentry provides an event filter above that maps events to types of sentry events, its `Layer`
+            // implementation does not participate in `tracing`'s level-fast-path implementation, which depends on
+            // a hidden api (<https://docs.rs/tracing-subscriber/latest/src/tracing_subscriber/layer/mod.rs.html#864-867>)
```

Comment:
Can you link to a permalink on GitHub, so this doesn't go stale the next time a new release of `tracing_subscriber` is released?